src/styles/generic/type.scss

```diff
@@ -0,0 +1,66 @@
+@import '../settings/global';
+@import '../settings/colors';
+@import '../settings/type';
+@import '../tools/mixins';
+@import '../tools/breakpoints';
+
+$HEADLINE_TEXT_SIZE: 18px;
+$HEADLINE_FONT_WEIGHT: 400;
+$PARAGRAPH_SPACING_MOBILE: 12px;
+$PARAGRAPH_SPACING: 16px;
+
+// http://csswizardry.com/2016/02/managing-typography-on-large-apps/
+h1, h2, h3, h4, h5, h6 {
+  font-size: $HEADLINE_TEXT_SIZE;
+  font-weight: $HEADLINE_FONT_WEIGHT;
+}
+
+// default font styles for paragraphs are covered by global body styles
+// see generic/page.scss
+p {
+  @include font($BODY_MOBILE);
+  @include create_margins($BODY_MOBILE);
+  @include create_paragraph_spacing($PARAGRAPH_SPACING_MOBILE);
+
+  @include bp(sm) {
+    @include font($BODY_TABLET);
+    @include create_margins($BODY_TABLET);
+  }
+
+  @include bp(lg) {
+    @include font($BODY);
+    @include create_margins($BODY);
+    @include create_paragraph_spacing($PARAGRAPH_SPACING);
+  }
+}
+
+// hr {
```

Comment:
remove if disused?

Reply:
oh I plan to use it. I just commented it out because I'm hitting a stopping point for the night.